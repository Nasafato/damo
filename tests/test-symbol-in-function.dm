print_int(15);

def test(): int {
    symbol x;

    x = 15;
    print_num(value(x));

    return 0;
}

test();
