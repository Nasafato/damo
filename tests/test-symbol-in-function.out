15
15.000000
